runtime/src/common/sgx/avr.rs

```diff
@@ -316,6 +324,22 @@ pub fn verify(avr: &AVR) -> Fallible<AuthenticatedAVR> {
         _ => return Err(AVRError::MalformedQuote.into()),
     };
 
+    #[cfg(target_env = "sgx")]
+    {
+        if option_env!("OASIS_UNSAFE_ALLOW_DEBUG_ENCLAVES").is_none() {
+            let af = match AttributesFlags::from_bits(u64::from_le_bytes(
+                quote_body.report_body.attributes[0..8].try_into()?,
```

Comment:
sorry for the scope creep: would it be possible to use sgx_isa to do the parsing of the whole report?

Reply:
done

Reply:
thanks!